04/app.js

```diff
@@ -0,0 +1,44 @@
+const array = [];
+
+function newArray(a,b) {
+  for (let i=0; i <= 20 - 1; i++) {
+    const randomNumbers = Math.floor(Math.random() * (b - a + 1) + a)
+    array.push(randomNumbers)
+  }
+  return array
+}
+
+const randomArray = newArray(5,40);
+console.log(randomArray)
+
+function getLargest(arr) {
+  arr.sort(function(a,b) {
+    return b - a;
+  });
+  arr.splice(0,10)
+  return array
+}
+
+const largestNumbers = getLargest(array);
```

Comment:
Ponieważ przypisujemy do zmiennej `randomArray` tablicę to ją w tym miejscu powinniśmy przekazać jako argument.